pkg/providers/instancetype/types.go

```diff
@@ -173,15 +173,16 @@ func computeCapacity(ctx context.Context, info *ec2.InstanceTypeInfo, amiFamily
 	blockDeviceMappings []*v1beta1.BlockDeviceMapping, kc *corev1beta1.KubeletConfiguration) v1.ResourceList {
 
 	resourceList := v1.ResourceList{
-		v1.ResourceCPU:              *cpu(info),
-		v1.ResourceMemory:           *memory(ctx, info),
-		v1.ResourceEphemeralStorage: *ephemeralStorage(amiFamily, blockDeviceMappings),
-		v1.ResourcePods:             *pods(ctx, info, amiFamily, kc),
-		v1beta1.ResourceAWSPodENI:   *awsPodENI(ctx, aws.StringValue(info.InstanceType)),
-		v1beta1.ResourceNVIDIAGPU:   *nvidiaGPUs(info),
-		v1beta1.ResourceAMDGPU:      *amdGPUs(info),
-		v1beta1.ResourceAWSNeuron:   *awsNeurons(info),
-		v1beta1.ResourceHabanaGaudi: *habanaGaudis(info),
+		v1.ResourceCPU:                *cpu(info),
+		v1.ResourceMemory:             *memory(ctx, info),
+		v1.ResourceEphemeralStorage:   *ephemeralStorage(amiFamily, blockDeviceMappings),
+		v1.ResourcePods:               *pods(ctx, info, amiFamily, kc),
+		v1beta1.ResourceAWSPodENI:     *awsPodENI(ctx, aws.StringValue(info.InstanceType)),
+		v1beta1.ResourceNVIDIAGPU:     *nvidiaGPUs(info),
+		v1beta1.ResourceAMDGPU:        *amdGPUs(info),
+		v1beta1.ResourceAWSNeuron:     *awsNeurons(info),
```

Comment:
Do we also want to support `aws.amazon.com/neurondevice`? From reading through the neuron docs, I can see that it looks like the `aws.amazon.com/neuron` resource only exists for backwards compatibility so we should support all 3 device categories: https://awsdocs-neuron.readthedocs-hosted.com/en/latest/containers/tutorials/k8s-setup.html#:~:text=Neuron%20device%20plugin%20exposes%20Neuron%20cores%20%26%20devices%20to%20kubernetes%20as%20a%20resource